WikipediaUnitTests/Code/MWKLanguageLinkControllerTests.m

```diff
@@ -142,7 +150,7 @@ - (void)testPreferredLanguageVariantForLanguageCode {
     XCTAssertEqualObjects(chineseResult, chineseLanguageVariantCode);
 
     // Test fallback not in app preferences or OS preferences
-    NSString *serbianResult = [self.controller preferredLanguageVariantCodeForLanguageCode:@"sr"];
+    NSString *serbianResult = [self.controller preferredLanguageVariantCodeForLanguageCode:@"uz"];
     XCTAssertEqualObjects(serbianResult, serbianLanguageVariantCode);
```

Comment:
Can you update the var names too? Like `serbianLanguageVariantCode` > `uzbekLanguageVariantCode`.

Reply:
Yes, sorry. I was so happy to get the test working that I missed that.

Reply:
I've made the change.